main.sh

```
@@ -0,0 +1,84 @@
+    exit 1
+fi
+
+# run the main python script
+#python /home/ubuntu/retrospective-update/retrospective-update/master.py
+#if [ $? -ne 0 ]; then
+#    echo "Error: Failed to run the routing script."
+#    exit 1
+#fi
+
+# empty the outputs and era5 directories
+rm -r $OUTPUTS_DIR/*
```

Shellcheck (0.10.0)

[warning] Use "${var:?}" to ensure this never expands to /* .

(SC2115)